src/components/tree-item/tree-item.tsx

```diff
@@ -165,6 +166,13 @@ export class TreeItem implements ConditionalSlotComponent {
     this.updateAncestorTree();
   }
 
+  componentDidrender(): void {
```

Comment:
Can you verify this get called? There's a typo in the lifecycle method.

Reply:
Good catch . This is not being invoked and animation is broken.